paasta_tools/instance/kubernetes.py

```diff
@@ -446,8 +446,12 @@ def kubernetes_status(
         kube_client=kube_client,
         namespace=job_config.get_kubernetes_namespace(),
     )
+    # For the purpose of active_shas/app_count, don't count replicasets that are at 0/0.
+    actually_running_replicasets = [
```

Comment:
I think this list is the inverse of what you want to include in the next line? e.g. these are all the 0/0 replicasets you want to ignore.